e2e-tests/testdata/echo_client/tsconfig.json

```diff
@@ -12,7 +12,7 @@
 
     /* Language and Environment */
     "target": "ESNext",                                  /* Set the JavaScript language version for emitted JavaScript and include compatible library declarations. */
-    "lib": [ "ES6", "DOM" ],                             /* Specify a set of bundled library declaration files that describe the target runtime environment. */
+    "lib": [ "ES6", "DOM", "ES2021.String" ],            /* Specify a set of bundled library declaration files that describe the target runtime environment. */
```

Comment:
added this as `replaceAll` is part of this version. I assume it shouldn't be a problem since we target `ESNext`